big-practice/src/js/services/api-service.service.js

```diff
@@ -1,19 +1,21 @@
-/**
- * Get by calling API
- */
-const getApi = (url, handleAfterSuccess) => {
-  // GET method implementation
-  const options = {
-    method: 'GET',
-  };
+export class ApiService {
+  /**
+  * Get by calling API
+  */
+  static async getApi(url) {
+    try {
+      // GET method implementation
+      this.options = {
+        method: 'GET',
+      };
 
-  fetch(url, options)
-    // Parses JSON response into native JavaScript objects
-    .then((response) => response.json())
-    .then((data) => { handleAfterSuccess(data); })
-    // Show error message when API call is wrong
-    .catch((error) => alert('Error! An error occurred.', error));
-};
+      this.res = await fetch(url, this.options);
```

Comment:
This code is wrong. Should not assign a response into class's variable

Reply:
updated!